ci/do-audit.sh

```diff
@@ -30,3 +30,7 @@ cargo_audit_ignores=(
   --ignore RUSTSEC-2023-0001
 )
 scripts/cargo-for-all-lock-files.sh audit "${cargo_audit_ignores[@]}" | $dep_tree_filter
+# avoid the pipeline swallows the error
+if [[ "${PIPESTATUS[0]}" -ne 0 ]]; then
+  exit 1
+fi
```

Comment:
```suggestion
(
  set +e
  scripts/cargo-for-all-lock-files.sh audit "${cargo_audit_ignores[@]}" | $dep_tree_filter
  # we want the `cargo audit` exit code, not `$dep_filter`'s
  exit ${PIPESTATUS[0]}
)
```

Reply:
I found that I commit a old suggestion. just pushed the latest version!